### PR TITLE
[ADD] migration script

### PR DESCRIPTION
**Pull request by hbrunn**
and a module that takes care of saving imported files (+migration)

This does a minor migration from account_banking and adds a module that can be used to restore 7.0's behavior of saving the imported file.

I do the PR against current 8.0, but it is meant to be applicable to #14, #15 and #19, I just want to PR this cleanly in order not to have to look through all the noise already to be reviewed in the other PRs

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The icon can be the sheet with a paperclip, that it's the usual symbol of attachments

**Comment by lmignon** [comment]
:+1:  (Code review) The top would be to have automated tests. (for the new module)

**Comment by hbrunn** [comment]
@lmignon good idea, here you go

File: account_bank_statement_import_save_file/models/account_bank_statement_import.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    _inherit = 'account.bank.statement.import'
+
+    @api.model
+    def _import_statement(self, statement):
+        (statement_id, notifications) = \
+            super(AccountBankStatementImport, self)._import_statement(
+                statement)
+        if statement_id:
+            # get raw file data from the stack
+            def get_data_file(frame):
```
Comment (nbessi) [inline account_bank_statement_import_save_file/models/account_bank_statement_import.py]: why the frame lookup don't we recive the file in the signature ?

Comment (hbrunn) [inline account_bank_statement_import_save_file/models/account_bank_statement_import.py]: back then, no. Now, it makes more sense to overwrite https://github.com/OCA/bank-statement-import/blob/8.0/account_bank_statement_import/account_bank_statement_import.py#L71

I'll put this to work in progress until i can follow up

**Comment by hbrunn** [comment]
@nbessi rebased on current upstream with the necessary changes

**Comment by nbessi** [comment]
:+1:

**Comment by hbrunn** [comment]
@pedrobaeza now with a new icon

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**